docs/API_Reference.md

```diff
@@ -50,51 +50,531 @@ Create a customer master key(CMK) for the user, which can be a symmetric or an a
-  | code | int | 200 | The result of the method call, 200 is success, others are fail|
-  | message | String | "success" | The description of result |
-  | cmk| String |cmk_base64":<br> ""AAAAAAAAAAAAAAAAAAAAAGCcKdP/fwAA***"}<br>  }  |The result in json object for the cmk which in based64 encoding |
+  | code | int | 200 | The result of the method call, 200 is success, others are fail. |
+  | message | String | "success" | The description of result. |
+  | cmk_base64 | String |"AAAAAAAAAAAAAAAAAAAAAGCcKdP/fwAA***" |The result in json object for the cmk which in based64 encoding. |
 
 - **Example**
 	- Request sample in python
-   ```python
+  ```python
     params = OrderedDict()
     params["appid"] = appid
     params["nonce"] = str(int.from_bytes(os.urandom(16), "big"))
     params["timestamp"] = int(time.time())
     params["sign"] = sign
     params["payload"] = {
-      "keyspec":"EH_RSA_3072",
-      "origin":"EH_INTERNAL_KEY"
+      "keyspec": "EH_RSA_3072",
+      "origin": "EH_INTERNAL_KEY"
     }
     requests.post(url="<ehsm_srv_address>/ehsm?Action=CreateKey", data=json.dumps(params), headers=headers)
-   ```
+  ```
 
 	- Response data
-	 ```python
-	Response= {
-    "code":200,
-    "message":"success!",
-    "result": {
-        "cmk_base64":"AAAAAAAAAAAAAAAAAAAAAGCcKdP/fwA***"
-     }
-     }
-    ```
+  ```python
+    Response= {
+        "code": 200,
+        "message": "success!",
+        "result": {
+            "cmk_base64":"AAAAAAAAAAAAAAAAAAAAAGCcKdP/fwA***"
+        }
+    }
+  ```
    *(return to the [Cryptographic Functionalities APIs](#eHSM-REST-API-Reference).)*
 ---
 ## Encrypt
+Encrypt an arbitrary set of bytes using the CMK.(only support symmetric types).
+
+- **Rest API format:**
+
+   POST <ehsm_srv_address>/ehsm?Action=Encrypt
+
+
+- **Request Payload:**
+
+  | Name | Type | Reference Value | Description |
+  |:-----------|:-----------|:-----------|:-----------|
+  | aad | String | "challenge" | Some extra datas input by the user, which could help to to ensure data integrity, and not be included in the cipherblobs. |
+  | cmk_base64 | String | "AAAAAAAAAAAAAAAAAAAAAGCcKdP/fwA***" | A symmetric cmk in BASE64 string. |
+  | plaintext | String | "plaintext***" | The datas of the plaintext. |
+
+  Notes: for the common request parameters, please refer to the [common params](#Common-Prameters)
+   
+- **Response Data:**
+
+  | Name | Type | Reference Value | Description |
+  |:-----------|:-----------|:-----------|:-----------|
+  | code | int | 200 | The result of the method call, 200 is success, others are fail. |
+  | message | String | "success" | The description of result. |
+  | ciphertext_base64 | String | "uSDos6NLWNVp4sQZS2+mzLvDw***" |The result in json object for the Ciphertext  which in based64 encoding. |
+
+- **Example**
+	- Request sample in python
+  ```python
+    params = OrderedDict()
+    params["appid"] = appid
+    params["nonce"] = str(int.from_bytes(os.urandom(16), "big"))
```

Comment:
nonce can be removed and the timestamp should also be updated.
Update for all the API examples.

Reply:
It will be fix.